src/config/dark-sites.config

```diff
@@ -267,6 +271,7 @@ hackersforcharity.org
 hackertyper.com
 hackforums.net
 hackthebox.eu
+hacktoberfest.digitalocean.com
```

Comment:
Not dark by default due to system color checking.

Reply:
Done